lib/tasks/admin.rake

```diff
@@ -54,4 +54,12 @@ namespace :admin do
   task openai: :environment do
     Openai.call
   end
+
+  desc "Test GetWorkdayFields"
+  task workday_fields: :environment do
+    # apply_url = 'https://motorolasolutions.wd5.myworkdayjobs.com/en-US/Careers/job/Software-Engineer_R47616/apply/'
+    # apply_url = 'https://adobe.wd5.myworkdayjobs.com/en-US/external_experienced/job/Sr-Machine-Learning-Engineer--Adobe-Firefly_R147574/apply'
+    apply_url = 'https://maxar.wd1.myworkdayjobs.com/en-US/MAXAR/job/Senior-Automation-Engineer_R21029/apply'
+    Importer::GetWorkdayFields.call(apply_url)
+  end
```

Comment:
Can you add one of these for each ATS integration Dan? 

Also can we add some validation condition(s) so that we know it's worked? e.g. Capybara checks all input fields are filled. Or checks a particular input field contains the value in AQS. Or we could deliberately not fill one of the required fields and click the submit button - if the form returns >1 error then there is an issue we haven't noticed

Reply:
Could also add similar functionality in the rspec folder in due course?

Reply:
Yep, the formfiller ones are all in the ff.rake file. I've been using the admin.rake file just for convenience of testing during development (restoring the file before pushing to main).

For classes that rely on Capybara, I'm not sure you want me to write rspec tests because they'll be slow. The rakefile works well for informal testing in development, where I give Capybara sleep instructions so I can verify visually that the form's correct.